src/seedu/addressbook/Main.java

```diff
@@ -84,9 +84,13 @@ private void runCommandLoopUntilExitCommand() {
         do {
             String userCommandText = ui.getUserCommand();
             command = new Parser().parseCommand(userCommandText);
-            CommandResult result = executeCommand(command);
-            recordResult(result);
-            ui.showResultToUser(result);
+            try {
+                CommandResult result = executeCommand(command);
+                recordResult(result);
+                ui.showResultToUser(result);
+            } catch (StorageOperationException e) {
+                ui.showToUser(e.getMessage());
```

Comment:
Just showing the error message is not considered as "good" exception handling.
Think how you can improve the user experience when such exceptions occur.

Reply:
Got it!